pages/docs/installation/installation-special-systems.md

```diff
@@ -16,6 +16,52 @@ The instructions may still be valuable for unlisted systems.
 
 ## Active systems
 
+### HAWK (HPE Apollo/AMD, Stuttgart)
+
+#### Building
+
+The following steps explain how to install preCICE on HAWK with PETSc and MPI (using the system standard MPE):
+(1) [Download Eigen](http://eigen.tuxfamily.org/index.php?title=Main_Page) and copy it to HAWK. Afterwards export the `EIGEN3_ROOT`, e.g.,
+
+```bash
+export EIGEN3_ROOT="$HOME/precice/eigen"
+```
+
+(2) Load available modules:
+
+```bash
+module load cmake boost petsc/<VERSION>-int32-shared
+```
+
+(3) Build preCICE. The library and include path for PETSc need to be defined explicitly (for an unknown reason):
+
+```bash
+cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX="my/install/prefix" -DPRECICE_PETScMapping=ON -DPETSc_INCLUDE_DIRS="$PETSC_DIR/include" -DPETSc_LIBRARIES="$PETSC_DIR/lib/libpetsc.so" -DPRECICE_PythonActions=OFF .. /path/to/precice/source
+
+make install -j 16
+```
+
+{% include important.html content="Running the tests for the installation will fail, even though the installation is fine, since the tests require MPI facilities which need to be enabled explicitly." %}
+
+#### Running on a single node
+
+Simulations on a single node are possible, but you explicitly need to specify the hardware. Otherwise the mpi jobs are executed on the same cores, which will slow down the whole simulation significantly. In order to run the a coupled simulation on a single node, the following command can be used:
```

Comment:
```suggestion
Simulations on a single node are possible, but you explicitly need to specify the hardware. Otherwise, the MPI jobs are executed on the same cores, which will slow down the whole simulation significantly. In order to run the a coupled simulation on a single node, use the following:
```

For the last change: even though this page is a bit inconsistent, we decided at some point to follow a "casual"/"active" language style in the documentation, as if explaining to a colleague over coffee.